berkeley-mobile/Fitness/Fitness+Controllers/GymClassesController.swift

```diff
@@ -62,8 +62,34 @@ extension GymClassesController: UITableViewDataSource {
             cell.eventCategory.backgroundColor = gymClass.color
             cell.eventCategory.isHidden = gymClass.class_type == nil
             cell.eventTaggingColor.backgroundColor = gymClass.color
+            
+            if gymClass.website_link != nil && gymClass.location == "Zoom" {
+                let tapGesture = EventTapGestureRecognizer(target: self, action:
+                                                            #selector(GymClassesController.zoomTapped(gesture:)))
+                tapGesture.eventUrl = gymClass.website_link!
```

Comment:
can we use if let url = gymClass.website_link instead of nil check then force unwrap